src/map.rs

```diff
@@ -155,6 +258,23 @@ impl<'a, K: Ord + 'a, V: 'a> ManagedMap<'a, K, V> {
         }
     }
 
+    pub fn range<'b, 'c, Q>(&'c self, key_begin: &Q, key_end: &Q) -> Range<'a, 'b, K, V>
+            where K: Borrow<Q>, Q: Ord + ?Sized, 'a: 'b, 'c: 'a
```

Comment:
Is `'c` really necessary here?

Reply:
Yes, the `ManagedMap` has to outlive the `Range`'s references.